src/_bentoml_impl/frameworks/unsloth/mapping.py

```diff
@@ -0,0 +1,49 @@
+RUNTIME_MAPPING = {
+    "llama": {
+        "service_config": {
+            "traffic": {"timeout": 300},
+            "resources": {"gpu": 1, "gpu_type": "nvidia-l4"},
```

Comment:
llama 8b and llama 70b will need different resources

Reply:
think that will be included in the example to change the resources for `build_bento`

Reply:
Since users can continue fine tune from a saved checkpoint, and we can only get model_type from config